src/containers/SignUpScreen/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spinner from 'react-native-loading-spinner-overlay';
+import { Text, Image } from 'react-native';
+import { connect } from 'react-redux';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+
+import SignUpForm from '../../components/user/SignUpForm';
+import { signUp } from '../../actions/userActions';
+import styles from './styles';
+
+const SignUpScreen = ({ signUp, loading = false }) => (
+  <KeyboardAwareScrollView
+    contentContainerStyle={styles.container}
+    resetScrollToCoords={{ x: 0, y: 0 }}
+  >
+    <Spinner visible={loading} />
+    <Image
+      style={styles.image}
+      source={require('../../assets/images/background.png')}
+      resizeMode={'stretch'}
```

Comment:
When you want to pass an string you can do it directly:

`resizeMode="stretch"`